ci_microshift.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+sudo yum install -y make golang
+
+./shellcheck.sh
+./microshift.sh
+
+# Run createdisk script
+# Set the zstd compression level to 10 to have faster
+# compression.
```

Comment:
There's a `--fast` option if the goal is to get fast compression. You could also drop `--long`?

Reply:
with fast the bundle size would be not that optimized so using this is kind of proper balance the size of bundle and time taken.